static/js/add-shows/new-show.js

```diff
@@ -41,8 +41,10 @@ MEDUSA.addShows.newShow = function() {
         sampleText += '</b>';
 
         // if we have a show name then sanitize and use it for the dir name
-        if (showName.length) {
-            $.get('addShows/sanitizeFileName', {name: showName}, function (data) {
+        if (showName.length > 0) {
```

Comment:
Exactly. I'd like to also move everything we have to a strict set of standards so it's always understandable and then write a doc staring out standards. This will also allow me to write a travis check with xo to only accept certain checks like this. 

If everything is always the same it's much easier to know why and where something breaks.